.eslintrc.json

```diff
@@ -0,0 +1,14 @@
+{
+  "env": {
+    "es6": true,
+    "node": true,
+    "browser": true
+  },
+  "rules": {
```

Comment:
You can also add error for console.log statements ( if we are already using logger )